spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  id { 1 }
+  provider { 'google_oauth2' }
+  uid { '100000000000000000000' }
+  email { 'john@example.com' }
```

Comment:
This will break future tests when we need to create multiple users 💣 all attributes must be randomized

Reply:
Similarly to https://github.com/nimblehq/ic-rails-huey-toby/pull/43#discussion_r1247512922, all attributes will be randomized by using Faker